### PR TITLE
Rename setting descriptions

### LINKED ISSUE #120559: Update enable/disable setting descriptions
Brought up [through user feedback](https://twitter.com/_tomalak/status/1378989217905266688) that our settings that mention "enabled/disable" in the description are confusing:

I realize that this is inconsistent with how we describe other settings and tend to use "Enable" only, to make that clear:

### PR DESCRIPTION
**Pull request by miguelsolorio**
This PR fixes #120559 and updates our setting descriptions to go from using "enable/disable" => "enables" and matches our other descriptions.

@rzhao271 @joaomoreno @mjbvz @sandy081 assigning to you since you were the last ones to touch these files (respectively).

### CODE REVIEW

File: extensions/json-language-features/package.nls.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	"json.schemas.fileMatch.desc": "An array of file patterns to match against when resolving JSON files to schemas. `*` can be used as a wildcard. Exclusion patterns can also be defined and start with '!'. A file matches when there is at least one matching pattern and the last matching pattern is not an exclusion pattern.",
 	"json.schemas.fileMatch.item.desc": "A file pattern that can contain '*' to match against when resolving JSON files to schemas.",
 	"json.schemas.schema.desc": "The schema definition for the given URL. The schema only needs to be provided to avoid accesses to the schema URL.",
-	"json.format.enable.desc": "Enable/disable default JSON formatter",
+	"json.format.enable.desc": "Enables the default JSON formatter",
 	"json.tracing.desc": "Traces the communication between VS Code and the JSON language server.",
 	"json.colorDecorators.enable.desc": "Enables or disables color decorators",
```
Comment (rzhao271) [inline extensions/json-language-features/package.nls.json:11]: This one's sneaky

Comment (misolori) [reply extensions/json-language-features/package.nls.json:11]: Wouldn't this still read the same?

- [ ] Enables the default JSON formatter == off
- [x] Enables the default JSON formatter == on

**Comment by misolori** [comment]
I'm going to hold off on this until we've nailed down a recommendation, continuing discussion in https://github.com/microsoft/vscode/issues/120559

File: extensions/json-language-features/package.nls.json (hunk 1 above)
Comment (rzhao271) [reply extensions/json-language-features/package.nls.json:11]: I meant line 11: `"json.colorDecorators.enable.desc": "Enables or disables...`

**Pull request closed without merging**